go.sum

```diff
@@ -147,8 +147,8 @@ github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d/go.mod h1:ma
 github.com/cpuguy83/go-md2man/v2 v2.0.0/go.mod h1:maD7wRr/U5Z6m/iR4s+kqSMx2CaBsrgA7czyZG/E6dU=
 github.com/creack/pty v1.1.7/go.mod h1:lj5s0c3V2DBrqTV7llrYr5NG6My20zk30Fl46Y7DoTY=
 github.com/cyphar/filepath-securejoin v0.2.2/go.mod h1:FpkQEhXnPnOthhzymB7CGsFk2G9VLXONKD9G7QGMM+4=
-github.com/datawire/ambassador v1.12.2-0.20210325164646-9d653a4f2cfa h1:sd5jtI+QMQQRghLh2zsEMTnp+iqnSc3jcbjkkh6hZ0U=
-github.com/datawire/ambassador v1.12.2-0.20210325164646-9d653a4f2cfa/go.mod h1:2grBLdYgILzrgTpenDMB5OeyhObIUaT+KwkLkZI1KDE=
+github.com/datawire/ambassador v1.12.3-0.20210401195424-3d91930ec3fd h1:yt9n2dnenlvOeTyOHuMKQRrFjFJNlMAURVrCiTUJjCM=
+github.com/datawire/ambassador v1.12.3-0.20210401195424-3d91930ec3fd/go.mod h1:2grBLdYgILzrgTpenDMB5OeyhObIUaT+KwkLkZI1KDE=
```

Comment:
Why? I'm not objecting, but I don't see anything else in the PR that looks like it would have prompted this.

Reply:
Ah I had to add the StatefulSet alias: https://github.com/datawire/ambassador/pull/3323